packages/squidex/schema/rules/rule4.json

```diff
@@ -1,13 +1,13 @@
 {
   "$schema": "./../__json/rule.json",
   "isEnabled": true,
-  "name": "Teams upated",
+  "name": "Teams Updated",
```

Comment:
hang on - did we have this rule already? i see it was added as part of [this PR](https://github.com/yldio/asap-hub/pull/1071/files#diff-c7331d70cf82ee31cdc53c03fd9f2e587bc72de2156c271e43c3083588ae16b7) - i take it it was added accidentally? how did it slip through the PR?

Reply:
yes, it was a mistake